### DISCUSSION TITLE
possible SSTI (Server Side Template Injection)

### DISCUSSION THREAD

**Discussion by SQRR** [pimcore/pimcore#17866] [Q&A / Support]
Hello everyone

Last night  we experience a SSTI attempt on one of our project.
We did some research and found out that you can inject a template when performing a POST request containing the var "template" on any published document.
This may trigger a Exception "Unable to find template" or any other Exception when there are missing params in the twig template.

Are you aware of it (in Pimcore 11 & 10)? And if what is your recommended way to deal with that?
Currently we registered an EventListener which is cleaning out any "template" or "controller" params from the request.

grateful for your response

**Comment by fashxp** [comment]
--> https://github.com/pimcore/pimcore/security/policy
